ReflectometryServer/motor_pv_wrapper.py

```diff
@@ -13,7 +23,26 @@ def __init__(self, pv_name):
         Creates a wrapper around a motor PV for accessing its fields.
         :param pv_name (string): The name of the PV
         """
-        self._pv_name = pv_name
+        self._pv_name = "{}{}".format(MYPVPREFIX, pv_name)
+        self._after_value_change_listeners = set()
+
+        rbv_pv = "{}.RBV".format(self._pv_name)
+        logger.debug("Monitoring {} for changes.".format(rbv_pv))
+        CaChannelWrapper.add_monitor(rbv_pv, self._trigger_after_height_change_listeners)
+
+    def _trigger_after_height_change_listeners(self, new_value, alarm_severity, alarm_status):
+        logger.debug("Triggered after height change listeners {}.".format(new_value))
```

Comment:
This is after any value change in a motor axis, does not have to be height - edit name and log entry to reflect this